electron-app/magnifier/rust-sampler-manager.test.ts

```
@@ -0,0 +1,519 @@
+import { ChildProcess } from 'child_process';
```

ESLint

[error] 'ChildProcess' is defined but never used.

(@typescript-eslint/no-unused-vars)